lookout/style/typos/benchmarks/typo_commits_report.py

```diff
@@ -4,13 +4,15 @@
 import logging
 from operator import itemgetter
 import os
+import pprint
 import time
-from typing import Any, Dict, Iterable, Iterator, List, Optional, Sequence, Tuple
+from typing import Any, Dict, Iterable, Iterator, List, Mapping, Optional, Sequence, Tuple
+from unittest.mock import patch
 
-from lookout.core.analyzer import ReferencePointer, UnicodeChange
+from lookout.core.analyzer import ReferencePointer
 from lookout.core.api.service_analyzer_pb2 import Comment
-from lookout.core.api.service_data_pb2 import File
-from lookout.core.data_requests import DataService, request_files
+from lookout.core.api.service_data_pb2 import Change
+from lookout.core.data_requests import _handle_analyze_rpc_errors, DataService, request_files
```

Comment:
We cannot import a protected function. We must make it public and release a new SDK version. Sad, but true.

Reply:
https://github.com/src-d/lookout-sdk-ml/pull/87